community/server/src/main/java/org/neo4j/server/AbstractNeoServer.java

```diff
@@ -130,6 +132,7 @@
     protected CypherExecutor cypherExecutor;
     protected WebServer webServer;
     protected Supplier<AuthManager> authManagerSupplier;
+    protected Supplier<UserManagerSupplier> userManagerSupplier;
```

Comment:
Why is this asymmetrical? We have a `Supplier<AuthManager>`, but not a `Supplier<UserManager>`?

It seems we are moving _away_ from a double-wrapper for `AuthManager` (changes below), but we are introducing it for `UserManager`.